src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -442,7 +443,9 @@ def _retrieve_links(self,
             for fqid in links_ids
         }
         for links_json in links.values():
-            links_json['content'] = json.loads(links_json['content'])
+            content = links_json['content']
+            reject('||' in content, '`||` is disallowed', content)
```

Comment:
Extract the joiner into a config attribute so you don't need to repeat it all over the place. There already is an attribute in the manifest service. Consolidate the two. We do want spaces around the joiner in the manifests but still disallow any occurrence of `||` with or without spaces.